doc/whats-new.rst

```diff
@@ -49,10 +49,10 @@ Breaking changes
   ``origin`` or ``offset`` parameters is recommended as a replacement for using
   the ``base`` parameter and using time offset arithmetic is recommended as a
   replacement for using the ``loffset`` parameter.
-
-
-Deprecations
-~~~~~~~~~~~~
+- The ``squeeze`` kwarg to ``groupby`` is completely deprecated. This has been the source of some quite confusing
```

Comment:
I might phrase this as "no longer supported" rather than "completely deprecated".